<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="box_start">
     <property name="title">
      <string>Start</string>
     </property>
     <layout class="QGridLayout" name="layout_start">
      <item row="0" column="0">
       <widget class="QCheckBox" name="check_autoLoadCatalogs">
        <property name="text">
         <string>Load catalogs when the program starts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="box_units">
     <property name="title">
      <string>Units</string>
     </property>
     <layout class="QGridLayout" name="layout_units">
      <item row="0" column="0">
       <widget class="QLabel" name="label_unit_frequency">
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_unit_frequency"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_unit_intensity">
        <property name="text">
         <string>Intensity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_unit_intensity"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_unit_temperature">
        <property name="text">
         <string>Temperature:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_unit_temperature"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="box_export">
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QGridLayout" name="layout_export">
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_lineEnding"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_lineEnding">
        <property name="text">
         <string>Line Ending:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="check_quoteText">
        <property name="text">
         <string>Quote Text</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="check_commentSign">
        <property name="text">
         <string>Start Title Line with ‘#’</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="check_includeTitle">
        <property name="text">
         <string>Include Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
